.github/ISSUE_TEMPLATE/cli-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: CLI Issue
```

Comment:
The name shows up in the title, so it should be more friendly.

I.e. "Bug report for the CLI"

Reply:
done

Reply:
done